sweep/sweeper.go

```diff
@@ -1665,6 +1665,14 @@ func (s *UtxoSweeper) monitorFeeBumpResult(set InputSet,
 			// in sweeper and rely solely on this event to mark
 			// inputs as Swept?
 			if r.Event == TxConfirmed || r.Event == TxFailed {
+				// Exit if the tx is failed to be created.
+				if r.Tx == nil {
```

Comment:
Isn't this an error with the even is TxConfirmed but no Tx is found ?

Reply:
yeah and it will be caught when we receive the bump event and validate it.